pages/How-to/guide/linking-issue-with-pr.md

```diff
@@ -1,4 +1,50 @@
-<div align="center">
-  <img src="https://user-images.githubusercontent.com/74750414/169188784-680014eb-ae5a-4062-b9c0-40276f28a261.png" />
-</div>
+# How to Link and Issue no with a PR ❓
+### Things to keep in 🧠
+> You can link up to **ten issues** to each pull request.
 
+> The issue and pull request must be in the **same repository**.
+
+## Linking an issue with a PR 🔗 
+1.  On GitHub.com, navigate to the main page of the repository.
+2.  Now [`Create a pull request`](https://opensource.pradumnasaraf.co/#/pages/How-to/guide/creating-pr) first.
+3.  And then [`Create an issue`](https://opensource.pradumnasaraf.co/#/pages/How-to/guide/rasing-issue).
+4. Under your repository name, click **Pull requests**.
+5. In the list of pull requests, click the pull request that you'd like to link to an issue.
+6. In the *right sidebar*, in the "**Development**" section click ⚙️.

```

Comment:
Instead of steps for issue and PR, we can assume the PR and issue is already created



__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

Reply:
Sure, updating the steps.